widget/src/components/messages/CarouselMessage.tsx

```diff
@@ -98,15 +99,19 @@ const CarouselMessage: React.FC<CarouselMessageProps> = ({
           <CarouselItem key={idx} message={message} idx={idx} />
         ))}
       </div>
-      <button
-        className="sc-message--carousel-control prev"
-        onClick={goToPrevious}
-      >
-        &#10094;
-      </button>
-      <button className="sc-message--carousel-control next" onClick={goToNext}>
-        &#10095;
-      </button>
+      {shouldDisplayNavigationButtons && (
+        <>
+        <button
+          className="sc-message--carousel-control prev"
+          onClick={goToPrevious}
+          >
+          &#10094;
+        </button>
+        <button className="sc-message--carousel-control next" onClick={goToNext}>
+          &#10095;
+        </button>
+        </>
```

Comment:
Thanks @abdou6666 , I think we have a linting issue here ? Or am I wrong ?

Reply:
You aren't wrong, it's fixed